crates/ruff_linter/resources/test/fixtures/ruff/RUF035.py

```diff
@@ -0,0 +1,56 @@
+# setup
+sep = ","
+no_sep = None
+
+# positives
+"""
+	itemA
+	itemB
+	itemC
+""".split()
```

Comment:
A few more examples would be great:

1. Where the left is an empty string: `"".split()`
1. An all whitespace string: `"     ".split()` ideally where the whitespace is a mixture of spaces, tabs, etc
1. Where the left contains no split points: `"/abc/".split()` (found in the ecosystem results)
1. Multiline with different indents: see https://github.com/scikit-build/scikit-build-core/blob/f6e60f41e46ce13ddbd344542d1bf45743b74514/tests/test_skbuild_settings.py#L440-L443 
1. Strings with unicode flag
1. Examples with comments in various positions 
   ```py
      ("a,b,c"
      # comment
      .split()
      )
   ```


We should also add examples for the following strings where I think it's okay if we don't support them

1. Raw strings
1. Implicit concatenated strings
1. Implicit concatenated strings with commented parts (where there are comments between the parts)
1. Implicit concatenated strings with different prefixes

Reply:
Added these cases, as well as `maxsize=-1` that I found people also use.

Reply:
Implicit concatenated strings are supported, but the fix is unsafe because of the comments. I could also exclude them from fixing and mark the fix as safe if you prefer.